### PR TITLE
fix #79134 eririo.club

### LINKED ISSUE #79134: eririo.club
### Issue URL (Anti Adblock Script)
 [https://eririo.club/8hJqS](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Feririo.club%2F8hJqS)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.23
Browser: | Opera
Stealth mode: | disabled
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base, <br/>Peter Lowe's Blocklist<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances, <br/>Adblock Warning Removal List<br/><br/><b>Language-specific:</b><br/> AdGuard Turkish, <br/>AdGuard Russian

### PR DESCRIPTION
**Pull request by wout-codes**
#79134

### CODE REVIEW

File: EnglishFilter/sections/antiadblock.txt [hunk 1]
```diff
@@ -871,7 +871,7 @@ tvchoicemagazine.co.uk#%#//scriptlet("set-constant", "showModal", "noopFunc")
 @@||ams.cdn.arkadiumhosted.com/advertisement/display/stable/display-ads.js$domain=games.metro.us
 games.metro.us#%#//scriptlet("adjust-setInterval", "game will", "", "0.02")
 ! https://github.com/AdguardTeam/AdguardFilters/issues/78293
-freshi.site,cookuve.com,srek.net,exe.app#%#//scriptlet("set-constant", "disableItToContinue", "noopFunc")
+eririo.club,freshi.site,cookuve.com,srek.net,exe.app#%#//scriptlet("set-constant", "disableItToContinue", "noopFunc")
```
Comment (zloyden) [inline EnglishFilter/sections/antiadblock.txt:874]: `@@||googletagmanager.com/gtag/js$domain=eririo.club` for cb platforms.
`freshi.site,cookuve.com,srek.net,exe.app` they use the same script, so add these domains too.

**Review by zloyden — CHANGES_REQUESTED** [review]

**Review by zloyden — CHANGES_REQUESTED** [review]
Also missed popups. `eririo.club#%#//scriptlet("abort-current-inline-script", "String.fromCharCode", "/emZnbG9hZGVk|break;case \$\.|break;case \d{1}/")` and HTML rule.

**Comment by wout-codes** [comment]
> Also missed popups. `eririo.club#%#//scriptlet("abort-current-inline-script", "String.fromCharCode", "/emZnbG9hZGVk|break;case \$\.|break;case \d{1}/")` and HTML rule.

I don't get any popups.

File: EnglishFilter/sections/content_blocker.txt [hunk 2]
```diff
@@ -1645,6 +1645,8 @@ sketchful.io#@#.adBanner
 @@||tileman.io^$generichide
 ! https://github.com/AdguardTeam/AdguardFilters/issues/59200
 @@||ask4movie.co^$generichide
+! https://github.com/AdguardTeam/AdguardFilters/issues/79134
+@@||googletagmanager.com/gtag/js$domain=eririo.club|freshi.site|cookuve.com|srek.net|exe.app
```
Comment (zloyden) [inline EnglishFilter/sections/content_blocker.txt:1649]: Also remove this
```
! https://github.com/AdguardTeam/AdguardFilters/issues/72650#issuecomment-771199740
@@||googletagmanager.com/gtag/js$domain=exe.app
```

**Review by zloyden — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/content_blocker.txt (hunk 2 above)
Comment (wout-codes) [reply EnglishFilter/sections/content_blocker.txt:1649]: What's wrong? You told to add all those domain at the first place.

**Comment by zloyden** [comment]
> I don't get any popups.

Don't worry, I have it. I can see their inline script and it's working.

File: EnglishFilter/sections/content_blocker.txt (hunk 2 above)
Comment (zloyden) [reply EnglishFilter/sections/content_blocker.txt:1649]: This is an old rule that will become unnecessary. Use the search.
https://github.com/AdguardTeam/AdguardFilters/blob/master/EnglishFilter/sections/content_blocker.txt#L1382

File: EnglishFilter/sections/content_blocker.txt [hunk 3]
```diff
@@ -1379,7 +1379,7 @@ genshin.gamedot.org#@#.adsbygoogle
 ! https://github.com/AdguardTeam/AdguardFilters/issues/74649
 freemc.host#@#.adsbygoogle
 ! https://github.com/AdguardTeam/AdguardFilters/issues/72650#issuecomment-771199740
```
Comment (zloyden) [inline EnglishFilter/sections/content_blocker.txt:1381]: Update the task link.

**Review by zloyden — CHANGES_REQUESTED** [review]

**Review by zloyden — APPROVED** [review]

**Pull request merged**